### DISCUSSION TITLE
The latest jamulus for old High Sierra OSX?

### DISCUSSION THREAD

**Discussion by openaspace** [jamulussoftware/jamulus#3172] [Technical Support & Help] [answered]
Hello I see the latest update doesn't compatible for High Sierra, there is a legacy version? 
thanks.

**Comment by jujudusud** [comment, marked as answer]
https://github.com/jamulussoftware/jamulus/releases/tag/r3_10_0

**Comment by ann0see** [comment]
Legacy is still available but deprecated. If possible you should upgrade your OS. We don't test macOS legacy anymore 
Edit: even ancient macs might be upgradable to a modern OS with OpenCore Legacy patcher. Having an SSD and some more RAM is beneficial.

**Reply by jujudusud** [reply]
Or install a Linux distribution on your mac hardware.
